fsd_utils/sqs_scheduler/task_executer_service.py

```diff
@@ -79,20 +79,22 @@ def _handle_message_receiving_and_processing(self):
                 self.visibility_time,
                 self.sqs_wait_time,
             )
-            self.logger.debug(f"{thread_id} Message Count [{len(sqs_messages)}]")
+            self.logger.debug("%s Message Count [%S]"), thread_id, {len(sqs_messages)}
```

Comment:
Is there a difference between `%s` and `%S`? 
My test throws an error with capital S.

Reply:
also good spot and probably be just fat-fingering 🥲